tests-lit/Makefile

```diff
@@ -23,7 +23,6 @@ test: ## Run integration tests
 CLEAN_FIND_CMD=find \
 	. \
 	-type f \( \
```

Comment:
We have a precompiled `llvm-11-macos-bitcode-sample.exe` file (04-bitcode-llvm-version-mismatch) that was deleted every time this command was run. Having `Output` folder included as `-name 'Output'` ensures that we delete all the `.exe` files from the `Output`.